exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>Rename <code>sc_extractor</code> to <code>src_extractor</code> to help understanding </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>